charts/dvpe-deployment-gloo/CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ## [Unreleased]
 
+## [3.3.0]
```

Comment:
This should be a major change shouldn't it?

Reply:
Hi @crimbler2 ,

It is not really a new feature. It is just a refactoring.
But I can increase the Version to 4.0.0 if you like.

Reply:
Yes please, this is breaking, as it requires a new operator to be installed in the cluster

Reply:
OK,

bumped version